src/pages/achievement/control/subcomponents/controlPanelTools/AchievementControlPanelTools.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { AchievementItem } from '../../../../../features/achievement/AchievementTypes';
+import AchievementInferencer from '../../../dashboard/subcomponents/utils/AchievementInferencer';
+import PrerequisiteAdder from './controlPanelUtils/PrerequisiteAdder';
+import PrerequisiteDeleter from './controlPanelUtils/PrerequisiteDeleter';
+import PrerequisitePositionEditor from './controlPanelUtils/PrerequisitePositionEditor';
+import TaskDeleter from './controlPanelUtils/TaskDeleter';
+import TaskPositionInserter from './controlPanelUtils/TaskPositionInserter';
+
+type AchievementControlPanelToolsProps = {
+  editableAchievement: AchievementItem;
+  setEditableAchievement: any;
+  inferencer: AchievementInferencer;
+  saveChanges: any;
+};
+
+function AchievementControlPanelTools(props: AchievementControlPanelToolsProps) {
```

Comment:
can de-structure props in the params, to have less variables defined

Reply:
don't think it makes too much of a difference though :X 

at most just 1 LOC lor XD

But thanks for the tip!

Reply:
haha true